src/pilot/install-director.sh

```diff
@@ -245,6 +228,12 @@ if [ -n "${overcloud_nodes_pwd}" ]; then
     run_command "virt-customize -a overcloud-full.qcow2 --root-password password:${overcloud_nodes_pwd} --selinux-relabel"
 fi
 
+if [ ${enable_powerflex} == 1 ]; then
```

Comment:
Should this be true?

Reply:
No, that's a shell script. We set the option to 1 if enable_powerflex is present.
        --enable_powerflex)
                enable_powerflex=1; shift 1;;